compiler/ccgexprs.nim

```diff
@@ -1839,6 +1839,9 @@ proc genSomeCast(p: BProc, e: PNode, d: var TLoc) =
     if srcTyp.kind in {tyPtr, tyPointer} and etyp.kind in IntegralTypes:
       putIntoDest(p, d, e, "(($1) (ptrdiff_t) ($2))" %
           [getTypeDesc(p.module, e.typ), rdCharLoc(a)], a.storage)
+    elif etyp.kind == tySequence:
```

Comment:
Please try

``elif p.config.selectedGc == gcDestructors and etyp.kind == tySequence:``

Reply:
Ah damn. Yeah I forgot that.